test/sharness/t0700-remotepin.sh

```diff
@@ -64,18 +88,9 @@ test_expect_success "output includes meaningful error" '
   test_cmp config_exp config_out
 '
 
-test_expect_success "'ipfs config show' doesn't include RemoteServices" '
-  ipfs config show > show_config &&
-  test_expect_code 1 grep RemoteServices show_config
-'
-
-test_expect_success "'ipfs config replace' injects remote services back" '
-  test_expect_code 1 grep -q -E "test_.+_svc" show_config &&
-  ipfs config replace show_config &&
-  test_expect_code 0 grep -q test_pin_svc "$IPFS_PATH/config" &&
-  test_expect_code 0 grep -q test_invalid_key_svc "$IPFS_PATH/config" &&
-  test_expect_code 0 grep -q test_invalid_url_path_svc "$IPFS_PATH/config" &&
-  test_expect_code 0 grep -q test_invalid_url_dns_svc "$IPFS_PATH/config"
```

Comment:
We need to have a test for `ipfs config replace` because it is a separate code path and is used by _Settings_ screen of WebUI at  http://127.0.0.1:5001/webui

Right now I am unable to edit config via WebUI – even if  unrelated values are edited (eg. port in `Addresses.Swarm`) I am unable to update config via `ipfs config replace` as it fails due to:

```
cannot show or change pinning services through this API (try: ipfs pin remote service --help)
```

Reply:
I took a pass at upgrading 'config replace'. See if it meets your needs.